microsoft-edge/webview2/concepts/browserfeatures.md

```diff
@@ -0,0 +1,178 @@
+---
+description: Feature differences between Microsoft Edge and WebView2
+title: Feature differences between Microsoft Edge and WebView2
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 03/08/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, WebView2, webview, wpf apps, wpf, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html
+---
+# Browser feature differences between Microsoft Edge and WebView2  
+
+WebView2 is based off of the Microsoft Edge browser. This enables developers to have the opportunity to extend features from the browser to WebView2-based apps, which can be very useful. However, WebView2 is not limited to browser-like apps. There are many scenarios in which the browser's features don't make sense or are unneccesary.    
+
+The purpose of this article is to provide the following information.  
+
+*   The current approach that the WebView2 team uses for browser features.  
+*   The status of features and the reasoning behind the state.  
+*   The ability for features to be turned on \(or turned off\).  
+
+## Guiding Principles  
+
+Browser features within the context of WebView2 should adheres to the following guidelines for most WebView2 features.  
+
+*   Most of the WebView2 features work the same as Microsoft Edge features.  The following exceptions define why a feature is turned off.  
+    *   It doesn't make sense within the context of WebView2.  
+    *   It has an extended service cost.  
+*   The WebView2 features shouldn't include any Microsoft Edge branding.  
+    
+## Features  
+
+The following list displays the WebView2 features that differ from the new Microsoft Edge browser.  
+
+About this table:
+- Feature: The name of the feature.
+- Default: The default experience on a new WebView2 instance
+- Configurable: If the feature can be toggled on/off via our API surface or command line switch.
+- Details: additional information relevant to the feature..
+
+| Feature | Default | Configurable | Details |  
+|:--- |:--- |:--- | :--- |  
+| Autofill for Addresses | On | Yes | This feature is turned on by default, you may turn it on or off via WebView2 Autofill API. |  
+| Autofill for Passwords | On | Yes | This feature is turned on by default, you may turn it on or off via WebView2 Autofill API. |  
+| Autofill for Payments | Off | No | This feature is turned off. |  
+| Browser Extensions | Off | No | This feature is turned off. |  
+| Browser Task Manager | Off | No | This feature is turned off. |  
+| Collections | Off | No | This feature is turned off. |  
+| Continue-where-I-left-off prompt | Off | No | This feature is turned off. |  
+| Downloads | On | Yes | WebView2 provides an api that allows you to customize the download UI to manipulate downloads, such as block, redirect, save, pause, and so on.  For more information, navigate to <!--[download api][Webview2ReferenceDownloadApi]-->. |  
```

Comment:
@david-risney - No, the idea was for this doc to **only** highlight the features that differ in webview2. I put downloads on here, because I thought our download experience would differ from the browser. Like we will probably disabled the side pane for downloads, and modify some of the buttons in the popup. I guess I can take the downloads section out for now, and add it back in when the new downloads UI is released. Thanks for calling this out.

Reply:
Also if its configurable via webview2 API (not command line) we should also add that feature to this doc, even if its default on.